### PR TITLE
Style up the home page

### PR DESCRIPTION
**Pull request by mcnuttandrew**

### CODE REVIEW

**Comment by mcnuttandrew** [comment]
styleup

I'm thinking of this PR less as a final iteration, and more as a step on the way to completion, once this lands we will be able to publish the new docs! Open to other's comments and suggestions on the styling.

**Comment by Apercu** [comment]
Looks good, but I think the banner is way to big, trying to fit the graphs into it might not be a good idea, we also loose some space on the right of the text.

Why not dispatch each graph next to each paragraph, like a placeholder thing. We could even alternate the order paragraph / graph left and right, might be interesting to try out.

File: docs/docs-app/components/header.js [hunk 1]
```diff
@@ -19,16 +19,21 @@ const renderTab = (tab, index) => (
...
 );
 
+const renderGithubLink = () => (
+  <a href="https://github.com/uber/react-vis" className="link">GITHUB</a>
+);
+
 export default class Header extends Component {
 
   render() {
+    const mappedTabs = tabs.map(renderTab).concat(renderGithubLink());
```
Comment (Apercu) [inline docs/docs-app/components/header.js]: Could be taken out of the render

File: docs/docs-app/demo.scss [hunk 2]
```diff
@@ -52,20 +54,38 @@ a:visited {
   display: flex;
 }
 
+.f-down {
```
Comment (Apercu) [inline docs/docs-app/demo.scss]: `.fcol` could be a better naming

**Comment by mcnuttandrew** [comment]
homepage

**Comment by jckr** [comment]
a word of caution - the colors of the default react-vis palette work best against a white background. 
2nd best against a very dark background. 
against a grey background, you'll always lose contrast so you'd have to compensate somehow. maybe use a slightly off-balance grey (ie more red or more blue or more green) and a complementary tint? 
against a colored background, some of these colors will definitely look weird.

**Pull request merged**